<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReGenESyS -- Reborn Generic and Expansible System Simulator</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionInput_Analyser"/>
    <addaction name="actionOutput_Analyser"/>
    <addaction name="actionScenario_Manager"/>
    <addaction name="actionOptimizaton"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Model</string>
    </property>
    <widget class="QMenu" name="menuSimulation_2">
     <property name="title">
      <string>Simulation</string>
     </property>
     <addaction name="actionStart"/>
     <addaction name="actionPause"/>
     <addaction name="actionStep"/>
     <addaction name="actionStop"/>
     <addaction name="separator"/>
     <addaction name="actionRun_Control"/>
    </widget>
    <addaction name="actionInformation"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_Model"/>
    <addaction name="menuSimulation_2"/>
    <addaction name="separator"/>
    <addaction name="actionReport"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionComponents"/>
    <addaction name="actionConsole"/>
    <addaction name="actionProperties"/>
    <addaction name="actionReports"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionSupport"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuArrange">
     <property name="title">
      <string>Arrange</string>
     </property>
     <addaction name="actionTop"/>
     <addaction name="actionBottom"/>
    </widget>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Factor"/>
    <addaction name="separator"/>
    <addaction name="actionGrid"/>
    <addaction name="actionSnap"/>
    <addaction name="separator"/>
    <addaction name="menuArrange"/>
    <addaction name="actionGroup"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuModel"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarModel">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBarEdit">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_Factor"/>
  </widget>
  <widget class="QToolBar" name="toolBarSimulator">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCheck_Model"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStep"/>
   <addaction name="actionStop"/>
   <addaction name="actionRun_Control"/>
  </widget>
  <widget class="QToolBar" name="toolBarHelp">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDocumentation"/>
   <addaction name="actionSupport"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetPlugins">
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Plugins</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsPlugins">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidgetPlugins">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>4</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetProperties">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>107</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsProperties">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tableWidgetProperties">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>200</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>100</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsConsole">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTextEdit" name="textEdit_Console">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack2/png/24x24/60.png</normaloff>:/rec/resources/icons/pack2/png/24x24/60.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New simulation model</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack2/png/24x24/52.png</normaloff>:/rec/resources/icons/pack2/png/24x24/52.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Close.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Save.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack2/png/24x24/86.png</normaloff>:/rec/resources/icons/pack2/png/24x24/86.png</iconset>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Door.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Door.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Undo.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Redo.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Cut.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Copy.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Paste.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Find.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/View.png</normaloff>:/rec/resources/icons/pack3/png/24x24/View.png</iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Zoom.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Zoom.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_Factor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Zoom.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Factor</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionSnap">
   <property name="text">
    <string>Snap</string>
   </property>
  </action>
  <action name="actionInput_Analyser">
   <property name="text">
    <string>Input Analyser</string>
   </property>
  </action>
  <action name="actionOutput_Analyser">
   <property name="text">
    <string>Output Analyser</string>
   </property>
  </action>
  <action name="actionScenario_Manager">
   <property name="text">
    <string>Scenario Manager</string>
   </property>
  </action>
  <action name="actionOptimizaton">
   <property name="text">
    <string>Optimizaton</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCheck_Model">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack2/png/24x24/58.png</normaloff>:/rec/resources/icons/pack2/png/24x24/58.png</iconset>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Play.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Play.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Next track.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Next track.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Stop.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionRun_Control">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Tune.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Tune.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionComponents">
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
  <action name="actionElements">
   <property name="text">
    <string>Components</string>
   </property>
  </action>
  <action name="actionModels">
   <property name="text">
    <string>Models</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Help book.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Help book.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Help.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Help.png</iconset>
   </property>
   <property name="text">
    <string>Support</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Help book 3d.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Help book 3d.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F1</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="text">
    <string>Bottom</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="actionComponents_2">
   <property name="text">
    <string>Components</string>
   </property>
  </action>
  <action name="actionElements_2">
   <property name="text">
    <string>Elements</string>
   </property>
  </action>
  <action name="actionTrace">
   <property name="text">
    <string>Trace</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="text">
    <string>Events</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/icons/pack3/png/24x24/Pause.png</normaloff>:/rec/resources/icons/pack3/png/24x24/Pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>Reports</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
